tests/test_rdataframe.py

```diff
@@ -0,0 +1,47 @@
+import unittest, ROOT
+from PyRDF import RDataFrame, RDataFrameException
+
+class RDataFrameConstructorTests(unittest.TestCase):
+
+    def test_integer_arg(self):
+        RDF = RDataFrame(10)
+
+        self.assertListEqual(RDF.args, [10])
+
+    def test_two_args(self):
+        RDF_1 = RDataFrame("treename", "file.root")
+
+        rdf_2_files = ["file1.root", "file2.root"]
+        RDF_2 = RDataFrame("treename", rdf_2_files)
+
+        reqd_vec = ROOT.std.vector('string')()
+        for elem in rdf_2_files:
+            reqd_vec.push_back(elem)
+
+        RDF_3 = RDataFrame("treename", reqd_vec)
+
+        self.assertListEqual(RDF_1.args, ["treename", "file.root"])
+        self.assertListEqual(RDF_2.args, ["treename", reqd_vec])
+        self.assertListEqual(RDF_3.args, ["treename", reqd_vec])
+
+    def test_three_args(self):
+        rdf_branches = ["file1.root", "file2.root"]
+        RDF_1 = RDataFrame("treename", "file.root", rdf_branches)
+
+        reqd_vec = ROOT.std.vector('string')()
+        for elem in rdf_branches:
+            reqd_vec.push_back(elem)
+
+        RDF_2 = RDataFrame("treename", "file.root", reqd_vec)
+
+        self.assertListEqual(RDF_1.args, ["treename", "file.root", reqd_vec])
+        self.assertListEqual(RDF_2.args, ["treename", "file.root", reqd_vec])
+
+    def test_incorrect_args(self):
```

Comment:
@etejedor @diocas Current RDataFrame also raises a `TypeError` when calling `RDataFrame()` with no arguments. Should it be also considered here?

Reply:
Sure

Reply:
Sounds good @etejedor and @JavierCVilla . I'll add more test cases :+1: